src/Core/Data/ClassifiedQuestion.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class ClassifiedQuestion
+	{
+
+		public UserProfile Author { get; set; }
+
+		public string Question { get; set; }
+
+		public ClassifiedAnswer Answer { get; set; }
+
```

Comment:
I think this should have an ID.  Since this is an EF object that will get represented as a table in the DB it'll need a foreign key ID to link it to the parent Classified.